bin/publish

```diff
@@ -46,12 +47,12 @@ if [ "${tag}" == "beta" ] && [[ ! "${branch}" =~ ^v.-beta$ ]] ; then
 fi
 
 # Update bump level
-if [ "${tag}" != "public" ] ; then
+if [ "${tag}" != "public" ] && [[ ! "${bump}" =~ ^${alphaVersion}* ]] ; then
   bump="pre${bump}"
 fi
 
 # alpha/dev get a timestamp in their version
-if [ "${tag}" == "dev" ] || [ "${tag}" == "alpha" ] ; then
+if [[ "${tag}" == "dev"  ||  "${tag}" == "alpha" ]] && [[ ! "${bump}" =~ ^${alphaVersion}* ]] ; then
   bump="${major}.${minor}.${patch}-${tag}.${id}"
```

Comment:
We already check for tag = alpha and create this alpha version here. So what is the use of `alphaVersion` variable here?